pyvista/plotting/utilities/sphinx_gallery.py

```diff
@@ -74,6 +76,56 @@ def _process_events_before_scraping(plotter):
         plotter.update()
 
 
+def generate_images(image_path_iterator: Iterator[str], dynamic: bool = False) -> List[str]:
+    """Generate images from the current plotters.
+
+    The file names are taken from the ``image_path_iterator`` iterator.
+
+    If a plotter has a ``_gif_filename`` attribute, a gif will be created.
+    Else, depending on the value of ``dynamic``, either a ``.png`` static image
+    or a ``.vtksz`` file will be created.
```

Comment:
```suggestion
    A gif will be created if a plotter has a ``_gif_filename`` attribute.
    Otherwise, depending on the value of ``dynamic``, either a ``.png`` static image
    or a ``.vtksz`` file will be created.
```